snapcraft/hooks/configure

```diff
@@ -33,9 +33,12 @@ get_bool() {
     return
 }
 
-# Don't fail if the mount namespace isn't properly setup yet
+# Try to fix an improperly setup mount namespace
 if [ ! -e /run/snapd-snap.socket ]; then
-    exit 0
```

Comment:
I wonder if we should just fail entirely, im not sure when the `configure` hook is called and why the snap socket shouldnt always be available.

Reply:
I'd like to try it, and see if it works, as otherwise it feels like it could be silently masking issues.